homeassistant/components/sharkiq/vacuum.py

```diff
@@ -84,6 +85,9 @@ def __init__(
         """Create a new SharkVacuumEntity."""
         super().__init__(coordinator)
         self.sharkiq = sharkiq
+        self._attr_name = sharkiq.name
```

Comment:
`name` and `serial_number` seem to be available right away in the library.
I wasn't so sure about `vac_model_number` / `oem_model_number` so didn't use the device_info attribute.